src/components/OrderItem.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+export class OrderItem extends React.Component {
+  action() {
+    this.props.removeFromOrder(this.props.index, this.props.price);
+  }
+
+  render() {
+    const { name, price, bread, isHot } = this.props;
+    return (
+      <li className="order__item">
```

Comment:
Could be a good point to have a new CSS component of `order-item`, quite a few cases where you are using the element as a new component `...__item-name`, `...__item-price` etc